src/main/java/games/strategy/engine/data/GameDataMemento.java

```diff
@@ -0,0 +1,168 @@
+import com.google.common.collect.Maps;
+
+import games.strategy.util.Version;
+import games.strategy.util.memento.MementoExporter;
+import games.strategy.util.memento.MementoImportException;
+import games.strategy.util.memento.MementoImporter;
+import games.strategy.util.memento.PropertyBagMementoExporter;
+import games.strategy.util.memento.PropertyBagMementoImporter;
+
+/**
+ * Provides factory methods for creating objects that can import mementos to and export mementos from game data.
+ */
+public final class GameDataMemento {
+  @VisibleForTesting
+  static final long CURRENT_SCHEMA_VERSION = 1L;
+
+  @VisibleForTesting
+  static final String SCHEMA_ID = "application/x.triplea.game-data-memento";
+
+  /**
+   * An immutable collection of default memento export options.
+   *
+   * <p>
+   * The key is the option name. The value is the option value.
+   * </p>
+   */
+  public static final Map<ExportOptionName, Object> DEFAULT_EXPORT_OPTIONS_BY_NAME = newDefaultExportOptionsByName();
+
+  private GameDataMemento() {}
+
+  @VisibleForTesting
+  static final class PropertyNames {
+    static final String NAME = "name";
+    static final String VERSION = "version";
+
+    private PropertyNames() {}
+  }
+
+  private static Map<ExportOptionName, Object> newDefaultExportOptionsByName() {
+    return Maps.immutableEnumMap(ImmutableMap.of(
+        ExportOptionName.EXCLUDE_DELEGATES, false));
+  }
+
+  /**
+   * Creates a new game data memento exporter using the default export options.
+   *
+   * @return A new game data memento exporter; never {@code null}.
+   */
+  public static MementoExporter<GameData> newExporter() {
+    return newExporter(DEFAULT_EXPORT_OPTIONS_BY_NAME);
+  }
+
+  /**
+   * Creates a new game data memento exporter using the specified export options.
+   *
+   * @param optionsByName The memento export options; must not be {@code null}. The key is the option name. The value is
+   *        the option value.
+   *
+   * @return A new game data memento exporter; never {@code null}.
+   */
+  public static MementoExporter<GameData> newExporter(final Map<ExportOptionName, Object> optionsByName) {
+    checkNotNull(optionsByName);
+
+    return new PropertyBagMementoExporter<>(SCHEMA_ID, CURRENT_SCHEMA_VERSION, new ExportHandlers(optionsByName));
+  }
+
+  /**
+   * Names the options that can be specified when creating a new game data memento exporter.
+   */
+  public static enum ExportOptionName {
+    /**
+     * Indicates delegates should be excluded from the memento (value type: {@code Boolean}).
+     */
+    EXCLUDE_DELEGATES
+  }
+
+  private static final class ExportHandlers implements PropertyBagMementoExporter.HandlerSupplier<GameData> {
+    private final Map<Long, PropertyBagMementoExporter.Handler<GameData>> handlersBySchemaVersion =
+        getHandlersBySchemaVersion();
+
+    // TODO: add support for options
+    @SuppressWarnings("unused")
+    private final Map<ExportOptionName, Object> optionsByName;
```

Comment:
Looking briefly, the presense of Maps to `Object` type is pretty concerning. If not just a marker interface, it seems we should be able to give the `Object` a type of some sort. Failing that, creating a custom data structure to wrap the Object and do casting for us as appropriate I've seen done to good success before. For example, the notation is getting a bit overwhelming at times, eg:

```
+    private Map<Long, PropertyBagMementoExporter.Handler<GameData>> getHandlersBySchemaVersion() {
 +      return ImmutableMap.<Long, PropertyBagMementoExporter.Handler<GameData>>builder()```

Does the concern make sense, would it make sense to fix it by creating 1st class data strucutre or using a stronger value type than `Object`?